custom/ilsgateway/views.py

```diff
@@ -328,3 +331,29 @@ def change_runner_date_to_last_migration(request, domain):
     last_run.end = checkpoint.date
     last_run.save()
     return HttpResponse('OK')
+
+
+class ReportRunListView(ListView, DomainViewMixin):
```

Comment:
Added in dispatcher: https://github.com/dimagi/commcare-hq/pull/7375/files#diff-774201133ac94fd25b946f8c1eb46c3cR341